### PR TITLE
WB-78 WB-71 WB-72 - Create fit file from activity data generated by SufferFestWeb,  Upgrade SDK, export data to NodeJS

### PR DESCRIPTION
**Pull request by jrmahechad**
#### What’s this PR do?
Create a .fit file base on a json object 
#### Any background context you want to provide?
- This update change the json structure base on the data available in SufferFestWeb
- The activity-from-data.js was generated with data from a real activity by SufferFestWeb
#### How should this be manually tested?

1. `npm install`
2. `npm test`
3. It should pass the test
4. Go to examples\content-to-js
5. `npm install`
6. `node index.js`
7. Verify that a file called temp.fit
8. Open the fit file in a desktop application like Golden Cheetah and review the content. (This file is created from NodeJS)

#### Could any automated tests be added for this?
Maybe
#### What are the relevant JIRA issues?
https://thesufferfest.atlassian.net/browse/WB-70
#### Open issues

### CODE REVIEW

File: examples/content-to-js/index.js [hunk 1]
```diff
@@ -11,14 +11,10 @@ const handleWritingFile = (err, binaryBuffer) => {
...
 
-const getBuffer = buffer => {
-  console.log("[Encode]", buffer);
-  const binaryBuffer = new Buffer(buffer, "binary");
-  fs.writeFile("temp.fit", binaryBuffer, err => handleWritingFile(err, binaryBuffer));
-};
 
 var buffer = fitParser.encode(activityObj);
 const binaryBuffer = new Buffer(buffer, "binary");
+console.log(typeof buffer,typeof binaryBuffer);
```
Comment (yawar-ali) [inline examples/content-to-js/index.js]: Can we remove this console.log ?

File: examples/buffer-example/index.js [hunk 2]
```diff
@@ -3,110 +3,97 @@ const addon = require('../../build/Release/binding.node');
...
-assert(addon.pass_integer(17.23) == undefined);
-assert(addon.pass_integer() == undefined);
 
-assert(addon.pass_boolean(false));
-assert(addon.pass_boolean(true) == false);
-assert(addon.pass_boolean() == undefined);
-assert(addon.pass_boolean("no") == undefined);
-assert(addon.pass_boolean(1) == undefined);
 
+const oauthHeader = 'oAuth oauth_consumer_key="54d00cdd-ee20-4dce-b111-4fcb39adbb37",oauth_token="d8f72b6d-07e2-45d2-bf4d-a6f55e10d5fc",oauth_signature_method="HMAC-SHA1",oauth_timestamp="1503953348",oauth_nonce="i0TIGJbWkfq",oauth_version="1.0",oauth_signature="O%2BK0K99gZ%2FnEJogerlLKXnM%2Fbjc%3D"';
```
Comment (yawar-ali) [inline examples/buffer-example/index.js]: I think we should be using ENV vars here.

File: examples/content-to-js/index.js (hunk 1 above)
Comment (jrmahechad) [reply examples/content-to-js/index.js]: ok, is removed

**Review by yawar-ali — APPROVED** [review]
Looks Good 👍

**Pull request merged**